gazebo_ros/test/test_sim_time.cpp

```diff
@@ -21,6 +21,8 @@
 #include <string>
 #include <vector>
 
+using namespace std::literals::chrono_literals; // NOLINT
```

Comment:
nitpick: the `NOLINT` disables this check, but usually we do two spaces between code and comments on the same line, e.g. `;··// NOLINT`